aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 
 ## Blazor
 
-We hope to have the Blazor section of this article updated soon.<!--In the meantime, see the following resources: -->
+<!-- UPDATE 8.0 Add API cross-links after RTM -->
 
-### Adopt .NET 8 features
+The following migration scenarios are covered:
 
-After following the guidance earlier in this article to update an app to 8.0, adopt specific features by following the links in <xref:aspnetcore-8#blazor>.
+* [Update a Blazor Server app](#update-a-blazor-server-app)
+* [Add interactive WebAssembly components to a Blazor Server app](#add-interactive-webassembly-components-to-a-blazor-server-app)
+* [Convert a Blazor Server app into a Blazor Web App](#convert-a-blazor-server-app-into-a-blazor-web-app)
```

Comment:
Let's merge this alternative approach to migration with the previous Blazor Server section.

```suggestion
```